Makefile.depend

```diff
@@ -0,0 +1,10 @@
+host2ip.o: host2ip.c sysdep.h
+hpt.o: hpt.c sysdep.h
+hsearch.o: hsearch.c sysdep.h hsearch.h
+multimer.o: multimer.c multimer.h notify.h sysdep.h
+notify.o: notify.c sysdep.h notify.h multimer.h ansi.h
+rd.o: rd.c rtpdump.h sysdep.h
+rtpdump.o: rtpdump.c rtp.h sysdep.h vat.h rtpdump.h ansi.h
+rtpplay.o: rtpplay.c sysdep.h notify.h rtp.h rtpdump.h multimer.h ansi.h
+rtpsend.o: rtpsend.c notify.h rtp.h sysdep.h multimer.h ansi.h
+rtptrans.o: rtptrans.c rtp.h sysdep.h rtpdump.h ansi.h notify.h multimer.h vat.h
```

Comment:
Does this mean that the list of compile-time dependencies is fixed? Are developers expected to keep this file synchronized with the source code?

Wouldn't it be better and more robust to let the compiler (gcc) generate the list of dependencies automatically?

Reply:
This list is generated by `mkdep`. It has not changed in a decade,
and can be redone with `make depend`

Reply:
I see. I think make depend is fine (as long as there is one command to update the dependency list).